TagListView.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "TagListView"
-  s.version      = "1.3.2"
+  s.version      = "1.3.1"
```

Comment:
I don't know if this affects the Expensify app, but I think it doesn't because Cocoapods check out on the tag. Just double check running `pod install` and see that your code still works on Expensify app.